### ISSUE TITLE
scons error 

### ISSUE THREAD

**Issue by justForAsk** [mitsuba-renderer/mitsuba#61]
I've tried to compile the mitsuba.

after the scons step I've got the following message:
Unable to detect a Qt installation -- not building the GUI!
Checking for cl ...no
Could not compile a simple C++ fragment, verify that cl is installed! This could also mean that the Boost libraries are missing. The file "config.log" should contain more information.

inside the config.log I've got the following message:
file C:\Users\mitsuba-master\build\SConscript.configure,line 137:
	Configure(confdir = .sconf_temp)
scons: Configure: Checking for cl ...
.sconf_temp\conftest_0.cpp <-
  |#include <sstream>
  | int main(int argc, char **argv) {
  | std::ostringstream oss;
  | return 0;
  | }
cl -o .sconf_temp\conftest_0.o -c /nologo /Od /Z7 /fp:fast /D WIN32 /D WIN64 /W3 /EHsc /GS- /GL /MD /D MTS_DEBUG /D SINGLE_PRECISION /D SPECTRUM_SAMPLES=3 /D MTS_SSE /D MTS_HAS_COHERENT_RT /D _CONSOLE /D DEBUG /D OPENEXR_DLL /openmp /D GLEW_MX -Idependencies\include\openexr -Idependencies\include\openexr -Iinclude -Idependencies\include .sconf_temp\conftest_0.cpp
conftest_0.cpp
cl : Command line warning D9035 : option 'o' has been deprecated and will be removed in a future release
link /nologo /SUBSYSTEM:CONSOLE /DEBUG /MACHINE:X64 /FIXED:NO /OPT:REF /OPT:ICF /LTCG /NODEFAULTLIB:LIBCMT /MANIFEST /OUT:.sconf_temp\conftest_0.exe /LIBPATH:dependencies\lib boost_system-vc141-mt-1_64.lib boost_filesystem-vc141-mt-1_64.lib boost_thread-vc141-mt-1_64.lib msvcrt.lib ws2_32.lib Half.lib zlib.lib .sconf_temp\conftest_0.o
LINK : fatal error LNK1181: cannot open input file 'boost_system-vc141-mt-1_64.lib'
scons: Configure: no

**Comment by merlinND** [comment]
Thanks for providing the `config.log` information. From what I can read, it looks like the dependencies directory might be missing. The required dependencies are provided for convenience in this repository:  
https://github.com/mitsuba-renderer/dependencies_win64

They should be placed at the root of your Mitsuba directory, with name `dependencies`.

**Comment by justForAsk** [comment]
Hey,

thanks for the reply, 
the dependencies are exist. 
I've installed scons-2.4.0.win-amd64.
system information: x64 
processor: intel(R) core(TM) i7-8700K CPU @ 3.7 GHZ, 3701 Mhz, 6 core(s), 12 logical processor(s)
I'm using Visual Studio 2017, let me know how to find my compiler details if its necessary
or post another information.

**Comment by merlinND** [comment]
Using the provided SCons 2.5.1 should help. Here are steps that worked for me. First, opening the "Developer Command Prompt for VS 2017" from the Start menu. This assumes a default Python2.7 installation (from the official website).

```
# You probably have done that already
git clone https://github.com/mitsuba-renderer/mitsuba.git
cd mitsuba
git clone https://github.com/mitsuba-renderer/dependencies_win64.git
cd dependencies

# Unzip scons-2.5.1 from the dependencies directory, then navigate to that directory
# Install provided scons
C:\Python27\python.exe setup.py install

# Now go back to the Mitsuba root directory and select a configuration file
cd ..
copy build\config-win64-msvc2017.py .\config.py

# Start compiling
C:\Python27\scons.bat 
```

Note that you may see "Unknown compiler version" messages, for this I will refer you to issue #52.

**Comment by justForAsk** [comment]
@merlinND , it works!
you are great, thank you very much.
indeed, I've got the message of "Unknown compiler version" and I fixed it following #52 
and now it builds it correctly.
However, I still have problem in building the gui, I've got the message:

```
`This application failed to start because it could not find or load the Qt platform plugin
''windows'' in '' '',

Available plattform plugins are: windows.

Reinstalling the application may fix the problem.

```
I reinstalled the Qt and scons again, I have 5.8 version of Qt.
I copied the "qwindows.dll" to "dist\platforms\qwindows.dll from the Qt
commented the unnecessary parts in qt5.py as @guoxx suggested in  issue #52.

What else I can do?
I'll ask also in #52, if you have any insights about it so I would like to knew.

thanks!
